java/pom.xml

```diff
@@ -198,8 +198,30 @@
                         <configuration>
                             <excludes>
                                 <exclude>**/CuFileTest.java</exclude>
+                                <exclude>**/CudaTest.java</exclude>
                             </excludes>
                         </configuration>
+                        <executions>
+                            <execution>
+                                <id>main-tests</id>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>fatal-cuda-test</id>
```

Comment:
The new executions for surefire only appear in the `no-cufile-test` profile which is disabled by default.  We need this in the main profile as well.

Reply:
Done.